docs-src/ref-programs-compute.scrbl

```diff
@@ -914,6 +934,28 @@ the value is not present in the array, @reachin{None} is returned.
 in the given array that satisfies the predicate @tt{f}. The return value is of type @reachin{Maybe(UInt)}. If
 no value in the array satisfies the predicate, @reachin{None} is returned.
 
+@subsubsection{@tt{Array.find} && @tt{.find}}
+
+@(mint-define! '("find"))
+@reach{
+  Array.find(arr, ty, f)
+  arr.find(ty, f) }
+
+@index{Array.find} @reachin{Array.find(arr, ty, f)} returns the first element of type, @reachin{ty}, in the array, @reachin{arr},
+that satisfies the predicate @reachin{f}. The return value is of type @reachin{Maybe(ty)}. If no value in the
+array satisfies the predicate, @reachin{None} is returned.
```

Comment:
It is unfortunate that we need to pass the type, given that it is stored in the array. I think these means there should be something like `arr.elemType` to read off the type as an `SLType`